### PR TITLE
STORY-7135 support passing invoca api oauth token in header for all request types

### PR DESCRIPTION
**Pull request by rjkearns**
add documentation/examples for specifying oauth token in the request headers

WEB [PR](https://github.com/Invoca/web/pull/10284)

### CODE REVIEW

**Review by jebentier — APPROVED** [review]

File: source/api_documentation/transactions_api/_intro.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Authentication
 --------------
 
-The API uses OAuth Authentication to authenticate that access is allowed. Pass the OAuth Token like any other query parameter, however, please note that the OAuth token is a required parameter.
+The API uses OAuth to validate access. The OAuth Token can be passed in two ways, the first being passed like any other query parameter. The second is to pass the OAuth Token in the header of the request. Please note that the OAuth Token is a required parameter.
```
Comment (collin-mcgrath) [inline source/api_documentation/transactions_api/_intro.rst]:
```suggestion
The API uses OAuth to validate access. The OAuth Token can be passed in two ways. The first way is to pass it like any other query parameter. The second is to pass the OAuth Token in the header of the request. Please note that the OAuth Token is a required parameter.
```

**Review by collin-mcgrath — CHANGES_REQUESTED** [review]
Requesting 1 grammatical change.

File: source/api_documentation/transactions_api/_intro.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@
 Authentication
 --------------
 
-The API uses OAuth Authentication to authenticate that access is allowed. Pass the OAuth Token like any other query parameter, however, please note that the OAuth token is a required parameter.
+The API uses OAuth to validate access. The OAuth Token can be passed in two ways. The first way is to pass it like any other query parameter. The second is to pass the OAuth Token in the header of the request. Please note that the OAuth Token is a required parameter.
```
Comment (nburwell) [inline source/api_documentation/transactions_api/_intro.rst]: should we flip the options around.. ie mention first way is to pass in header of request (I assume that will be a new preferred best practice as it keeps it out of the url for GET requests)

**Review by nburwell — COMMENTED** [review]
So excited to see this item worked!

File: source/api_documentation/transactions_api/_intro.rst (hunk 2 above)
Comment (rjkearns) [reply source/api_documentation/transactions_api/_intro.rst]: Yeah, that's a good call. Let me flip this around. Thanks for catching that.

Comment (rjkearns) [reply source/api_documentation/transactions_api/_intro.rst]: done

**Review by nburwell — APPROVED** [review]

**Review by collin-mcgrath — APPROVED** [review]

**Comment by rjkearns** [comment]
@nburwell would you be able to merge this?

**Pull request merged**